### PR TITLE
A couple more parameters for your service

### PR DESCRIPTION
**Pull request by ferrigno**
Hi, it would be useful to have two more pars in your service, I added them.

### CODE REVIEW

File: dump_ihklc.c [hunk 1]
```diff
@@ -209,7 +210,7 @@ int acsdump_xyz2radec(double x, double y, double z, double *ra, double *dec, dou
  }
 
 
-inline int RILerror(int status,MessageType mt, const char* format, ... ) {
+int RILerror(int status,MessageType mt, const char* format, ... ) {
```
Comment (volodymyrss) [inline dump_ihklc.c:214]: why remove inline?

Comment (volodymyrss) [reply dump_ihklc.c:214]: Ok well it has little impact and might ease building in some cases.

**Comment by volodymyrss** [comment]
Thanks, very useful!

**Comment by ferrigno** [comment]
It did not compile on ubuntu 20.04, that's why I removed the "inline"

**Pull request merged**